src/main/flight/gps_rescue.c

```diff
@@ -326,21 +346,15 @@ static void performSanityChecks()
 
     previousTimeUs = currentTimeUs;
 
-     // Stalled movement detection
-    static int8_t gsI = 0;
-
-    gsI = constrain(gsI + (rescueState.sensor.groundSpeed < 150) ? 1 : -1, -10, 10);
+    secondsStalled = constrain(secondsStalled + (rescueState.sensor.groundSpeed < 150) ? 1 : -1, -10, 10);
 
-    if (gsI == 10) {
+    if (secondsStalled == 10) {
```

Comment:
This is definitely not counting seconds. Maybe `stallIndicator`?

Reply:
It looks like secondsStalled and secondsLowSats are updated once per second, because of this code:

    const uint32_t currentTimeUs = micros();
    const uint32_t dTime = currentTimeUs - previousTimeUs;

    if (dTime < 1000000) { //1hz
        return;
    }

Reply:
Ok, I stand corrected. Requiring 20 seconds of no movement sounds like a long time.